content/influxdb/v2.0/write-data/delete-data.md

```diff
@@ -0,0 +1,104 @@
+4. In the **Delete Data** window that appears:
+  - Select a **Target Bucket** to delete data from.
+  - Enter a **Time Range** to delete data from.
+  - Click **+ Add Filter** to filter by tag key and value pair.
+  - Select **I understand that this cannot be undone**.
+5. Click **Confirm Delete** to delete the selected data.
+
+### Delete data from the Data Explorer
+
+1. Click the **Data Explorer** icon in the sidebar.
+
+    {{< nav-icon "data-explorer" >}}
+
+2. Click **Delete Data** in the top navigation bar.
+3. In the **Delete Data** window that appears:
+  - Select a **Target Bucket** to delete data from.
+  - Enter a **Time Range** to delete data from.
+  - Click **+ Add Filter** to filter by tag key-value pairs.
+  - Select **I understand that this cannot be undone**.
+4. Click **Confirm Delete** to delete the selected data.
+!-->
+
+Use the `influx` CLI or the InfluxDB API `/delete` endpoint to delete data.
+
+{{% note %}}
+The `-p, --predicate` flag is supported in InfluxDB Cloud and InfluxDB OSS 2.0 beta 16 or earlier.
+
+In InfluxDB OSS 2.0rc0, the `influx delete --predicate` flag has been disabled.
+{{% /note %}}
+
+## Delete data using the influx CLI
+
+1. Use the [`influx delete` command](/influxdb/v2.0/reference/cli/influx/delete/) to delete points from InfluxDB.
+2. Specify your organization, bucket, and authentication token.
+3. Define the time range to delete data from with the `--start` and `--stop` flags.
+4. Specify which points to delete using the `--predicate` or `-p` flag and
+   [Delete predicate syntax](/influxdb/v2.0/reference/syntax/delete-predicate/).
+
+##### Example delete command
+```sh
+influx delete -o my-org -b my-bucket -t $INFLUX_TOKEN \
+  --start '1970-01-01T00:00:00.00Z' \
+  --stop '2020-01-01T00:00:00.00Z' \
+  --predicate '_measurement="sensors" and sensorID="abc123"'
+```
+
+{{% warn %}}
+Running `influx delete` without the `-p` or `--predicate` flag deletes all data with
+timestamps between the specified `--start` and `--stop` times in the specified bucket.
+{{% /warn %}}
+
+## Delete data using the API
+1. Use the InfluxDB API `/delete` endpoint to delete points from InfluxDB.
+2. Include your organization and bucket as query parameters in the request URL.
+3. Use the `Authorization` header to provide your InfluxDB authentication token.
+4. In your request payload, define the time range to delete data from with `start` and `stop`.
+   Specify which points to delete using the `predicate` and
+   [Delete predicate syntax](/influxdb/v2.0/reference/syntax/delete-predicate/).
+
+##### Example delete request
+```sh
+curl -XPOST http://localhost:8086/api/v2/delete/?org=myOrg&bucket=myBucket \
+  -H 'Authorization: Token <YOURAUTHTOKEN>' \
+  -d '{
+        "start": "1970-01-01T00:00:00.00Z",
+        "stop": "2020-01-01T00:00:00.00Z",
+        "predicate": "_measurement=\"sensors\" and sensorID=\"abc123\""
+      }'
+```
```

Comment:
### Example delete requests

**InfluxDB OSS 2.0rc0** does not support the `predicate` parameter.

#### Delete data in InfluxDB Cloud

```sh
curl -X POST 'https://us-west-2-1.aws.cloud2.influxdata.com/api/v2/delete?orgID=<ORGID>' 
-H 'Authorization: Token <TOKEN WITH WRITE PERMISSIONS' 
-H 'Content-Type: application/json' 
-d '{
"predicate": "_measurement=\"<MEASUREMENT NAME>\" and _field=\"<FIELD>\"",
"start": "2020-08-16T08:00:00Z",
"stop": "2020-08-17T08:00:00Z"
}'
```

#### Delete data in InfluxDB OSS

```sh
curl -XPOST http://localhost:8086/api/v2/delete/?org=myOrg&bucket=myBucket \
-H 'Authorization: Token <YOURAUTHTOKEN>' \
-d '{
"start": "1970-01-01T00:00:00.00Z",
"stop": "2020-01-01T00:00:00.00Z"
}'
```